packages/react-charts/src/components/ChartUtils/chart-tooltip.ts

```diff
@@ -49,7 +50,12 @@ export const getCursorTooltipCenterOffset = ({
   offsetCursorDimensionY = false,
   theme
 }: ChartCursorTooltipCenterOffsetInterface) => {
-  const pointerLength = theme && theme.tooltip ? theme.tooltip.pointerLength : 10;
+  const pointerLength =
+    theme && theme.tooltip
+      ? isFunction(theme.tooltip.pointerLength)
+        ? theme.tooltip.pointerLength({ index: 0 })
```

Comment:
Although only `index` appears to be required for `CallbackArgs`, we should also provide `data`, `datum`, `x`, and, `y` since they are available.

Reply:
Note that `index` is a valid prop for this component, so we should pass in `rest.index` first.